snippets/typescript-snippets.cson

```diff
@@ -18,4 +18,36 @@
              * $1
              */
         """
- 
+    'exportinterface':
+            'prefix': 'exi'
+            'body': """
+                export interface ${1:IName}{${2:value:number;}}
+            """
+    'exportclass':
+        'prefix': 'exc'
+        'body': """
+            export class ${1:Name}{
+            this._init:boolean;
+            constructor(public idx:number, init:boolean){
+                  _init=init;
+                  }
+            memberFunction(){
+
+            }
+          }
+        """
```

Comment:
thanks for understanding. `exp` does autocomplete to export. `cla` autocomplete's to `class`